src/backend/marsha/markdown/api.py

```diff
@@ -34,8 +45,11 @@ class MarkdownDocumentViewSet(
     serializer_class = serializers.MarkdownDocumentSerializer
 
     permission_classes = [
-        core_permissions.IsTokenResourceRouteObject
-        & (core_permissions.IsTokenInstructor | core_permissions.IsTokenAdmin)
+        (
+            core_permissions.IsTokenResourceRouteObject
+            & (core_permissions.IsTokenInstructor | core_permissions.IsTokenAdmin)
+        )
+        | markdown_permissions.IsMarkdownDocumentPlaylistOrOrganizationAdmin
```

Comment:
Generic question : Should the playlist `created_by` also give permission to add a Markdown document (or any resource) to the playlist? Or we expect the `models.PlaylistAccess` administrator level to be created along when the user creates a new playlist? (because I guess the playlist creator must have admin access to its own playlist)

Reply:
Interesting point.
I think we should implement your second proposal.
WDYT @lunika ?

Reply:
Yes I think we should create administrator access in the same time a user creates a playlist.